src/Robo/Blt.php

```diff
@@ -116,6 +120,60 @@ private function addPluginsCommandsAndHooks() {
   }
 
   /**
+   * Adds Symfony (non-Robo) command classes to the application.
+   *
+   * @param \Acquia\Blt\Robo\Application $application
+   */
+  protected function addSymfonyCommands(Application $application) {
+    $twig = new Environment(new FilesystemLoader());
+
+    // Get any custom defined Twig filters to be ignored by linter.
+    $twig_filters = (array) $this->getConfig()->get('validate.twig.filters');
+    // Add Twig filters from Drupal TwigExtension to be ignored.
+    // @todo Find a way so that this list doesn't need to be maintained.
```

Comment:
Yeah, I think this is where I started to lose the thread. I wanted to somehow load `Drupal\Core\Template\TwigExtension` into the `$twig` environment, but that led down a rabbit hole of determining how to get the right services to inject into `TwigExtension`.